src/components/sbb-form-field/sbb-form-field.tsx

```diff
@@ -0,0 +1,213 @@
+import { InterfaceSbbFormFieldAttributes } from './sbb-form-field.custom';
+import { AgnosticMutationObserver as MutationObserver } from '../../global/helpers/mutation-observer';
+
+let nextId = 0;
+
+/**
+ * @slot label - Slot to render a label.
+ * @slot prefix - Slot to render an icon on the left side of the input.
+ * @slot input - Slot to render an input.
+ * @slot suffix - Slot to render an icon on ther right side of the input.
+ * @slot error - Slot to render an error.
+ */
+@Component({
+  shadow: true,
+  styleUrl: './sbb-form-field.scss',
+  tag: 'sbb-form-field',
+})
+export class SbbFormField {
+  /**
+   * Add a specific space if the `<sbb-error>` is present.
+   */
+  @Prop() public errorSpace?: InterfaceSbbFormFieldAttributes['errorSpace'] = 'default';
+
+  /**
+   * Add a `<label>` for the input.
+   */
+  @Prop() public label: string;
+
+  /**
+   * Indicates whether the input is optional.
+   */
+  @Prop() public optional?: boolean;
+
+  /**
+   * Size variant, either l or m.
+   */
+  @Prop() public size?: InterfaceSbbFormFieldAttributes['size'] = 'l';
+
+  /** Whether to display the form field without a border. */
+  @Prop() public borderless = false;
+
+  /**
+   *
+   * It is used internally to get the native `disabled` attribute from `<input>`.
+   */
+  @State() private _disabled: boolean;
+
+  /**
+   *
+   * It is used internally to get the native `readonly` attribute from `<input>`.
+   */
+  @State() private _readonly: boolean;
+
+  /**
+   * It is used internally to get the `prefix` slot.
+   */
+  @State() private _prefixElements: Element[] = [];
+
+  /**
+   * It is used internally to get the `suffix` slot.
+   */
+  @State() private _suffixElements: Element[] = [];
+
+  /**
+   * It is used internally to get the `suffix` slot.
+   */
+  @State() private _errorElements: Element[] = [];
+
+  /** Whether the input inside the form field is invalid. */
+  @State() private _invalid = false;
+
+  private _originalInputAriaDescribedby?: string;
+
+  /**
+   * It is used internally to get the `input` slot.
+   */
+  private _input: HTMLInputElement | HTMLSelectElement | HTMLElement;
+
+  /**
+   * Listen the changes on `readonly` and `disabled` attributes of `<input>`.
+   */
+  private _formFieldAttributeObserver = new MutationObserver((mutations: MutationRecord[]) =>
+    this._onAttributesChange(mutations)
+  );
+
+  public disconnectedCallback(): void {
+    this._formFieldAttributeObserver.disconnect();
+  }
+
+  /**
+   * It is used internally to set the aria-describedby attribute for the slotted input referencing available <sbb-form-erro> instances.
+   */
+  private _onSlotErrorChange(event: Event): void {
+    this._errorElements = (event.target as HTMLSlotElement).assignedElements();
+    const value = this._errorElements.length
+      ? this._errorElements.map((e) => e.id).join(',')
+      : this._originalInputAriaDescribedby;
+    this._input.setAttribute('aria-describedby', value);
```

Comment:
Can we maybe prevent console errors with that?
```suggestion
    this._input?.setAttribute('aria-describedby', value);
```